osgi-run-core/src/main/groovy/com/athaydes/gradle/osgi/bnd/BndWrapper.groovy

```diff
@@ -25,7 +25,7 @@ class BndWrapper {
 
         Map<String, Object[]> config = getWrapConfig( wrapInstructions, jarFile )
         def consumeValue = { String key ->
-            Object[] items = config.remove( key )
+            Object[] items = config.get( key )
```

Comment:
This is not correct because we need to pass on any config entries we did not remove to the Bnd Analyser. So you need to make a copy of the config Map first, then remove from that copy.